lib/solidus_paypal_braintree/engine.rb

```diff
@@ -54,6 +54,7 @@ def self.activate
           config.menu_items << config.class::MenuItem.new(
             [:braintree],
             'cc-paypal',
+            label: 'braintree',
```

Comment:
If a label is not specified, it will use the [first section](https://github.com/solidusio/solidus/blob/9ae2ed16bad7e29ea422fab1403118a3a0e66266/backend/app/models/spree/backend_configuration.rb#L58), which in this case is `:braintree`, that label [will be passed into the `tab` method](https://github.com/solidusio/solidus/blob/master/backend/app/views/spree/admin/shared/_tabs.html.erb#L7) which [just translate it](https://github.com/solidusio/solidus/blob/9ae2ed16bad7e29ea422fab1403118a3a0e66266/backend/app/helpers/spree/admin/navigation_helper.rb#L56). Not sure why this change is needed honestly.